### ISSUE TITLE
Add terms-of-service to directory

### ISSUE THREAD

**Issue by bifurcation** [ietf-wg-acme/acme#59]
Right now, in order to discover the CA's subscriber agreement, you have to create a new registration and get it back in a Link header.  Since this is a static property of the CA, it should be possible to get it without taking any action.  The directory is the natural place for it.

**Comment by mholt** [comment]
I like the idea of a meta field in the directory, as mentioned on the mailing list:

```json
"meta": {
    "terms-of-service": "https://example.com/acme/subscriber-agreement-v01.pdf",
    "website": "https://www.example.com/",
},
```

**Comment by bifurcation** [comment]
Fixed by #72
